lib/_emerge/show_invalid_depstring_notice.py

```diff
@@ -22,18 +19,26 @@ def show_invalid_depstring_notice(parent_node, error_msg):
         pkg_location = os.path.join(
             parent_node.root_config.settings["EROOT"], portage.VDB_PATH, category, pf
         )
-        msg.append("Portage is unable to process the dependencies of the ")
-        msg.append("'%s' package. " % p_key)
-        msg.append("In order to correct this problem, the package ")
-        msg.append("should be uninstalled, reinstalled, or upgraded. ")
-        msg.append("As a temporary workaround, the --nodeps option can ")
-        msg.append("be used to ignore all dependencies.  For reference, ")
-        msg.append("the problematic dependencies can be found in the ")
-        msg.append("*DEPEND files located in '%s/'." % pkg_location)
+        msg.append(
+            (
```

Comment:
They are not. It was from habit of trying to make string literal concatenation more "explicit" in most cases because of prior feedback from other developers in the past.